src/main/java/org/jenkinsci/confluence/plugins/StatsInfoParser.java

```diff
@@ -45,7 +45,7 @@
             // y axis - data (number of installations)
             "&chd=t:{2}" +
             // grid lines
-            "&chg=10,-1,0,0" +
+            "&chg=9.09,-1,0,0" +
```

Comment:
Or we could let Google figure it out with `chg=-1,-1,0,0` with the same result, but good enough I think.